features/GEM005_geometry-decomposed.objects.feature

```diff
@@ -0,0 +1,31 @@
+@informal-proposition
+@gem
+Feature: GEM005 - Constraints on representation identifiers
+
+"""
+    In cases of aggregation of physical elements into a physical aggregate
+    the shape representation of the whole (within the same representation identifier)
+    must be taken from the sum of the shape representations of the parts.
+
+    If a building element acts as container, i.e. has parts associated,
+    and those parts have own shape representations, then the container
+    must have no shape representations other than 'Body'.
+"""
+  Background: Entity is part having own shape representation
+
+    Given An IfcBuildingElement
```

Comment:
In a constant attempt to tease implementers IfcBuildingElement in IFC4X3 onwards has been renamed to IfcBuiltElement. So we either need to have a Given that selects on schema version and duplicate the scenario or do something clever with the entity selection.